web/api/v0/__init__.py

```diff
@@ -449,6 +449,11 @@ def get(self):
 
         person_id = get_swdd_person_id(args.first_name, args.last_name, args.birthdate)
 
+        # some tenants have an additional semicolon added to their last names
+        if person_id is None:
+            person_id = get_swdd_person_id(
+                args.first_name, args.last_name + ";", args.birthdate)
```

Comment:
```suggestion
            person_id = get_swdd_person_id(
                args.first_name, args.last_name + ";", args.birthdate
            )
```
If the opening bracket is followed by a line break, the closing bracket should be preceded by one. Other than that it's perfectly fine, thank you!

Reply:
Hi @lukasjuhrich 

I applied the changes.

(Though, even the PEP8 standard is a bit unclear about this. You can find
```python
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```
as well as 
```python
result = some_function_that_takes_arguments(
    'a', 'b', 'c',
    'd', 'e', 'f',
)
```
in there.)